webapp/pages/profile/_id/_slug.vue

```diff
@@ -167,7 +167,7 @@
               tag="h5"
               color="soft"
             >
-              Wer folgt {{ user.name | truncate(15) }}?
+              Wer folgt {{ userName(user.name,15) }}?
```

Comment:
```suggestion
              Wer folgt {{ truncateUserName(user.name,15) }}?
```
If find this more telling. The method does not return the userName, it's truncating it. (+ having a user-specific default)

Reply:
resorve by seperation